Using "Link to" to Phone correction doesn't work

Observation at workshop today: When using phone correction, the "link to" way of building relationships doesn't seem to work. I think this is because the Phone correction component has multiple input fields. But a default should be selected IMO.

Stack trace in log file:

```
INFO  15:48:21.832 [AWT-EventQueue-0] JobGraphLinkPainter - Failed to add input columns (14) to TransformerComponentBuilder[transformer=Phone Correction,inputColumns=[TransformedInputColumn[id=trans-0028-0029,name=id (token 1)]]]
java.lang.IllegalStateException: Could not assign value '[Lorg.datacleaner.api.InputColumn;@5e2084f3' to org.datacleaner.api.InputColumn com.hi.phone.datacleaner.PhoneTransformer.phoneNumberColumn
	at org.datacleaner.descriptors.AbstractPropertyDescriptor.setValue(AbstractPropertyDescriptor.java:84)
	at org.datacleaner.job.builder.AbstractComponentBuilder.setConfiguredPropertyIfChanged(AbstractComponentBuilder.java:351)
	at org.datacleaner.job.builder.AbstractComponentBuilder.setConfiguredProperty(AbstractComponentBuilder.java:288)
	at org.datacleaner.job.builder.AbstractComponentBuilder.addInputColumn(AbstractComponentBuilder.java:465)
	at org.datacleaner.widgets.visualization.JobGraphLinkPainter.createLink(JobGraphLinkPainter.java:239)
	at org.datacleaner.widgets.visualization.JobGraphLinkPainter.endLink(JobGraphLinkPainter.java:150)
	at org.datacleaner.widgets.visualization.JobGraphMouseListener.graphReleased(JobGraphMouseListener.java:257)
	at edu.uci.ics.jung.visualization.control.MouseListenerTranslator.mouseReleased(MouseListenerTranslator.java:87)
	at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
	at java.awt.AWTEventMulticaster.mouseReleased(Unknown Source)
	at java.awt.Component.processMouseEvent(Unknown Source)
	at javax.swing.JComponent.processMouseEvent(Unknown Source)
	at java.awt.Component.processEvent(Unknown Source)
	at java.awt.Container.processEvent(Unknown Source)
	at java.awt.Component.dispatchEventImpl(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
	at java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
	at java.awt.Container.dispatchEventImpl(Unknown Source)
	at java.awt.Window.dispatchEventImpl(Unknown Source)
	at java.awt.Component.dispatchEvent(Unknown Source)
	at java.awt.EventQueue.dispatchEventImpl(Unknown Source)
	at java.awt.EventQueue.access$400(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.awt.EventQueue$3.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.awt.EventQueue$4.run(Unknown Source)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(Unknown Source)
	at java.awt.EventQueue.dispatchEvent(Unknown Source)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.pumpEvents(Unknown Source)
	at java.awt.EventDispatchThread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: Can not set org.datacleaner.api.InputColumn field com.hi.phone.datacleaner.PhoneTransformer.phoneNumberColumn to [Lorg.datacleaner.api.InputColumn;
	at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(Unknown Source)
	at sun.reflect.UnsafeFieldAccessorImpl.throwSetIllegalArgumentException(Unknown Source)
	at sun.reflect.UnsafeObjectFieldAccessorImpl.set(Unknown Source)
	at java.lang.reflect.Field.set(Unknown Source)
	at org.datacleaner.descriptors.AbstractPropertyDescriptor.setValue(AbstractPropertyDescriptor.java:82)
	... 40 more
```

--- a/desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphLinkPainter.java
+++ b/desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphLinkPainter.java
@@ -236,7 +236,6 @@ public class JobGraphLinkPainter {
                         componentBuilder.addInputColumns(getRelevantSourceColumn(sourceColumns, inputProperty),
                                 inputProperty);
                     } else {
-//                        componentBuilder.addInputColumn(sourceColumns.get(0), inputProperty);
                         componentBuilder.setConfiguredProperty(inputProperty, sourceColumns.get(0));
                     }
                     _actions.showConfigurationDialog(componentBuilder);

--- a/desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphLinkPainter.java
+++ b/desktop/ui/src/main/java/org/datacleaner/widgets/visualization/JobGraphLinkPainter.java
@@ -236,7 +236,8 @@ public class JobGraphLinkPainter {
                         componentBuilder.addInputColumns(getRelevantSourceColumn(sourceColumns, inputProperty),
                                 inputProperty);
                     } else {
-                        componentBuilder.addInputColumn(sourceColumns.get(0), inputProperty);
+//                        componentBuilder.addInputColumn(sourceColumns.get(0), inputProperty);
+                        componentBuilder.setConfiguredProperty(inputProperty, sourceColumns.get(0));
                     }
                     _actions.showConfigurationDialog(componentBuilder);
 

